ClosedXML/Excel/Ranges/XLRangeBase.cs

```diff
@@ -725,15 +729,19 @@ public XLCell LastCellUsed(Boolean includeFormats, Func<IXLCell, Boolean> predic
                         if (!cell.IsEmpty(true)) return cell;
                     }
                 }
+            }
 
-                var intersectedRanges = Worksheet.MergedRanges.GetIntersectedRanges(RangeAddress).ToList();
-                if (intersectedRanges.Any())
-                {
-                    Int32 minRo = intersectedRanges.Max(r => r.RangeAddress.LastAddress.RowNumber);
-                    Int32 minCo = intersectedRanges.Max(r => r.RangeAddress.LastAddress.ColumnNumber);
+            var intersectedRanges = Worksheet.MergedRanges.GetIntersectedRanges(RangeAddress)
```

Comment:
Same here.

Reply:
PS: In some places Visual Studio actually recommends to use the `predicate?.Invoke()` syntax.